src/components/Restoration/YourCompanySummary.vue

```diff
@@ -36,13 +60,18 @@
           </v-col>
         </v-row>
       </div>
+      <div class="section-container">
```

Comment:
Should this be conditional (ie, show only if it has changed)?

Reply:
I don't think so. The current implementation of component is as below (screenshots)
If it not changed:
![image](https://user-images.githubusercontent.com/116035339/229867368-43aaa2bc-ccda-40d6-a8ad-223da6e56b96.png)

If changed, show changed label:
![image](https://user-images.githubusercontent.com/116035339/229867535-86eb1c24-bc13-4f31-b581-ba190f20351b.png)

Reply:
OK, that looks good. Thanks for confirming :)